src/project/ProjectManager.js

```diff
@@ -369,6 +377,34 @@ define(function (require, exports, module) {
     };
 
     /**
+     * Moves the item in the oldPath to the newDirectory directory
+     *
+     * See `ProjectModel.moveItem`
+     */
+    ActionCreator.prototype.moveItem = function(oldPath, newDirectory) {
+        var self = this;
+
+        // Remove selected marker
+        self.setSelected(null);
```

Comment:
As of now, I just remove the selected flag if any of the items are moved. Need feedback on the correct way to handle it.